src/jetson/models/BlazeFace/blazeface.py

```diff
@@ -156,27 +178,38 @@ def forward(self, x):
 
     def _device(self):
         """Which device (CPU or GPU) is being used by this model?"""
```

Comment:
can you document what is exactly being returned here? just reading it I don't know what is so I don't think others will as well.